person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permession
```

Comment:
- [ ] please note that the `:id should only be able to be retrieved so only the Getter is defined` , kindly update your code by adding `attr_reader :id` 🙏